src/components/coffeechat/CoffeeChatModal/CoffeeChatModal.tsx

```diff
@@ -0,0 +1,242 @@
+import RHFControllerFormItem from '@/components/common/form/RHFControllerFormItem';
+import Input from '@/components/common/Input';
+import Loading from '@/components/common/Loading';
+import useAlert from '@/components/common/Modal/useAlert';
+import useCustomConfirm from '@/components/common/Modal/useCustomConfirm';
+import Responsive from '@/components/common/Responsive';
+import Text from '@/components/common/Text';
+import TextArea from '@/components/common/TextArea';
+import { MB_BIG_MEDIA_QUERY,MB_MID_MEDIA_QUERY,MOBILE_MEDIA_QUERY } from '@/styles/mediaQuery';
+import { zIndex } from '@/styles/zIndex';
+
+
+
+const schema = yup.object().shape({
+  phone: yup.string()
+  .nullable()
+  .required('연락처를 입력해주세요.')
+  .matches(PHONE_REGEX_SHORT, "'-'없이 입력해주세요"),
+  content: yup.string().required('내용을 입력해주세요.').max(500, '500자 이내로 입력해주세요.'),
+});
+
+const COFFEECHAT_PLACEHOLDER =
+  `ex. 안녕하세요, 커리어 전환을 고민 중인 PM 원하연이라고 합니다. 
+\n
+1. 최근 주변 PM들의 넥스트 커리어는 어떻게 되는지 궁금합니다.`;
+
+interface MessageForm {
+  phone: string;
+  content: string;
+}
+
+interface MessageModalProps {
+  phone: string;
+  name: string;
+  receiverId: string;
+  onClose:()=>void
+}
+
+const MessageModal: FC<MessageModalProps> = ({
+  receiverId,
+  phone,
+  name,
+  ...props
+}) => {
+  
+  const {
+    handleSubmit,
+    control,
+    formState: { isValid: _isValid },
+  } = useForm<MessageForm>({
+    resolver: yupResolver(schema),
+    mode: 'onChange',
+  });
+  const isValid = _isValid;
+  const { mutateAsync, isPending } = usePostMemberMessageMutation();
+
+  const { confirm, ConfirmComponent } = useCustomConfirm();
+  const { open } = useToast();
+
+  const submit = async ({ content, phone }: MessageForm) => {
+    if(isPending){
+      return;
+    }
+      const result = await confirm({
+        title: '커피챗 제안을 보내시겠습니까?',
+        description: '작성하신 내용은 상대방에게 문자로 전달돼요. 한번 보낸 커피챗 제안은 취소할 수 없어요.',
+        okButtonColor: colors.white,
+        okButtonTextColor: colors.black,
+        okButtonText: '보내기',
+        cancelButtonText: '취소',
+        zIndex: zIndex.헤더+102,
+        width: 400,
+      });
+    try {
+      if (result) {
+        await mutateAsync({
+          senderPhone: phone,
+          content,
+          receiverId,
+          category:"커피챗"
+        });
+        open({
+            icon: 'success',
+            content: '커피챗 제안이 잘 전달되었어요!',
+            style: {
+              content: {
+                whiteSpace: 'pre-wrap',
+              },
+            },
+          });
+          props.onClose();
+      }
+    } catch (error) {
+      throw error;
+    }
+  };
+
+  return (
+    <StyledModal isOpen {...props}>
+      <StyledForm onSubmit={handleSubmit(submit)}>
+        <ProfileImage src="./icons/icon_coffeechat.svg"/>
+        <Text mt={30} typography='SUIT_24_B'>
+          커피챗 제안하기
+        </Text>
+        <Text style={{"textAlign":"center","lineHeight":"22px"}} mt={14} typography='SUIT_14_M' color={colors.gray300}>
```

Comment:
p1: 
```suggestion
        <Text style={{ textAlign: 'center', lineHeight: '22px' }} mt={14} typography='SUIT_14_M' color={colors.gray300}>
```
여기 인라인스타일로 된 부분 모두 따옴표 없어야할 것 같은데!!! 혹시 저장할 때 자동으로 안바뀌나요??